forem/forem
# Cypress tests for `/admin/config` Authentication Section

## What type of PR is this? (check all applicable)

- [X] Writing Tests

## Description
This PR adds cypress tests for the Authentication Section user-flow in `/admin/config`.

## Related Tickets & Documents
This PR is a followup to [#12135](https://github.com/forem/forem/pull/12135)

## QA Instructions, Screenshots, Recordings
[You need Cypress installed on your system]
1. Pull branch down. In one tab, run Elasticsearch; in a second tab, run `yarn e2e` and choose `yes` when asked if to rebuild/reset test databases.
2. In the Cypress console, click on `authenticationSection.spec.js`. The test browser should popup. Ensure that all tests pass.

https://www.loom.com/share/bc28dec7df424aa8828193c438c6dbb1

### UI accessibility concerns?
None

## Added tests?

      have not been included_

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?
None

## [optional] What gif best describes this PR or how it makes you feel?

![Marge Simpson testing a mic](https://media.giphy.com/media/3orieKKmYyvUdR3RkY/giphy.gif)
